Debug/Scripts/gdb_uefi.py

```diff
@@ -42,6 +42,8 @@
 
 
 class ReloadUefi(gdb.Command):
+    # pylint: disable=undefined-variable
```

Comment:
I believe you are best to disable undefined-variable and no-self-use warnings globally on this file.